### DISCUSSION TITLE
Integrating gated/paywalled plugin?

### DISCUSSION THREAD

**Discussion by dreaktr** [Tetrakern/fictioneer#57] [Ideas]
I've used 'authcred' by @iniznet on my madara-wp themed site before. 

Any chances if you and iniznet would collab together to integrate some kind of similar features in Fictioneer?

**Comment by iniznet** [comment]
I personally don’t think I’ll make them plug-and-play without owners putting in some work for free, unless Tetrakern wish so.

However, you can integrate the authcred plugin into Fictioneer using the available hooks. I’ve made a few pull requests to introduce new hooks that help Fictioneer support the plugin or adapt it to your needs. Of course, you’ll still have to code and customize it to make the paywall in Fictioneer work as expected, or at least in line with authcred’s view on how paywalled chapters should work.

I’d recommend hiring Tetrakern to get it done for you, as he’s invested countless hours into developing this great, completely free story/novel theme for everyone.

**Comment by Tetrakern** [comment]
Thank you, I was thinking about how to break that news. Yes, I will not and neither should @iniznet do this for free. There is open source and there is slaving yourself away.
